webpack/ForemanWebhooks/Routes/Webhooks/Components/WebhookForm/Components/AutocompleteInput.js

```diff
@@ -0,0 +1,283 @@
+import React, { useEffect, useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Select,
+  SelectOption,
+  SelectList,
+  MenuToggle,
+  TextInputGroup,
+  TextInputGroupMain,
+  TextInputGroupUtilities,
+} from '@patternfly/react-core';
+import { sprintf, translate as __ } from 'foremanReact/common/I18n';
+
+const DEFAULT_PLACEHOLDER = __('Start typing to search');
+export const AutocompleteInputComponent = ({
```

Comment:
Don't we already have some sort of autocomplete input somewhere else that we could reuse?

Reply:
What about https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/SearchBar/SearchAutocomplete.js ?